### DISCUSSION TITLE
Quick change to use cases page on apache.org website

### DISCUSSION THREAD

**Discussion by jarrickvancamp** [apache/pulsar#23238] [General]
Hi on the page "use cases" on apache.org under the section "Scalable RPC" the third item in the list reads: 

> Networking rules to define _**which defines**_ which service can access which network

Any other place to report these small mistakes? Let me know

**Comment by lhotari** [comment]
Thank you @jarrickvancamp. 

The issue tracker is currently https://github.com/apache/pulsar/issues.
The task for improving the instructions for reporting of website issues was never taken to completion, that's https://github.com/apache/pulsar/issues/22277 . :( 

Would you like to contribute the fix directly to https://github.com/apache/pulsar-site? That's much appreciated.

For example, you could edit https://github.com/apache/pulsar-site/blob/main/data/usecases.ts#L30 directly in GitHub UI and create a pull request.

**Reply by jarrickvancamp** [reply]
Thanks, I will fork and create a pull request.

Cheers
